storage/src/tests/bucketmover/bucketmovertest.cpp

```diff
@@ -137,7 +121,7 @@ BucketMoverTest::testMaxPending()
 
     std::vector<api::StorageMessage::SP> msgs = after->getCommandsOnce();
     // 5 is the max pending default config.
-    CPPUNIT_ASSERT_EQUAL(5, (int)msgs.size());
+    EXPECT_EQ(5, (int)msgs.size());
```

Comment:
All checks for message vector size should preferably be `ASSERT_EQ` instead of `EXPECT_EQ` since there are some assumptions on what vector positions are valid after the check is done

Reply:
Fixed